src/Assert.php

```diff
@@ -1137,6 +1157,8 @@ public static function unicodeLetters($value, $message = '')
     }
 
     /**
+     * @psalm-assert string $value
```

Comment:
`ctype_*` functions have this fun behaviour where an integer between `-128` and `256` is converted to their respective char on the ASCII table. other integers are cast to a string. Everything else will silently fail.

So there is the possibility this will not be a string.

Reply:
Would `string|int` work here then?

Reply:
I think that is the correct annotation

Reply:
This won't work yet, due to https://github.com/vimeo/psalm/issues/2452

Will add to the patch for now, knowing that it will fail until upstream issue has some sort of resolution (which may also be to remove the type assertion here)